build/Test.targets

```diff
@@ -89,7 +89,7 @@
   </Target>
 
   <Target Name="TestsForBuildItself">
-    <DotNetMSBuild Arguments="/v:diag $(RepoRoot)/test/MsBuildScript.Tests/runtests.target"
+    <DotNetMSBuild Arguments="/v:diag $(RepoRoot)/test/MsBuildScript.Tests/runtests.target /p:PB_AssetRootUrl=$(PB_AssetRootUrl)"
```

Comment:
Is it only the asset root url that needs to be passed here?

Reply:
As long as this works, great, but it may be worth wrapping in escaped quotes in the command to prevent weird escaping.

Reply:
@mmitche 
PB_PACKAGEVERSIONPROPSURL and EXTERNALRESTORESOURCES have already generated the 'nuget.config' and the 'OrchestratedPackageVersionsProps.props' - however, 'PB_AssetRootUrl' is needed to re-calculate the download locations for the secondary MSBuild call.